src/main/java/org/broadinstitute/barclay/argparser/CommandLineArgumentParser.java

```diff
@@ -69,6 +69,9 @@
     public static final String COMMENT = "#";
     public static final String POSITIONAL_ARGUMENTS_NAME = "Positional Argument";
 
+    // Extension for collection argument list files
+    private static final String COLLECTION_LIST_FILE_EXTENSION = ".list";
```

Comment:
Also agree its not discoverable, and that we should do something, but the same is true for using "null" to reset collections, regular arguments files, comments, etc.. The usage output is already intimidating, and thats just for PrintReads! I wonder if we should maybe add a link to an online help document about using the CLP ? @vdauwera any opinions on that ?